### DISCUSSION TITLE
Home page with .blades

### DISCUSSION THREAD

**Discussion by alex129** [statamic/cms#7156] [General]
Hello!

I had to install statamic to an existing laravel proyect. I need to use it with .blades.php files so I was trying to test the collections and blueprints but I couldn't cause laravel show the error:

![image](https://user-images.githubusercontent.com/10452211/205106402-86516cc7-723e-41a2-b37a-91cf6ab8359a.png)

The problem is that with the installation in the existing laravel proyect I don't have any collection and blueprint

![image](https://user-images.githubusercontent.com/10452211/205106683-0dc4377e-bc92-4986-a999-9e90c2bd8b15.png)

My controller:

![image](https://user-images.githubusercontent.com/10452211/205107174-6bb01bc3-c459-4bdf-b816-9b83bb04e89c.png)

My route:

![image](https://user-images.githubusercontent.com/10452211/205107259-a2f79643-29ef-46e8-a91e-e16c769fc65e.png)

How can I create the home page and use the $page variable in my blade template?

Thanks!

**Comment by jasonvarga** [comment]
If you don't have any collections, then there'd be no page/entry to put in the `$page` variable. What are you expecting?

**Reply by alex129** [reply]
Hello Jason,

I already tryied to create a collection, for example "pages" but the error is still shown.

I'd like to know how can I create a collection for my home page with .blade.php and use $page variable to get title and content.

**Reply by jasonvarga** [reply]
If you've created the entry, in your controller you can retrieve it and pass it to your view.


```php
return view('home.home', [
  'page' => \Statamic\Facades\Entry::find($id)
]);
```

or you could do `Entry::findByUri('/')`.

**Reply by alex129** [reply]
It doesn't work, I've created a collection called Pages with a simple Entry Called Home

![image](https://user-images.githubusercontent.com/10452211/205254138-2877d387-07b3-411b-99fc-8cc660a80011.png)

I tought that the $page variable was automatically sent to the vew. But I can't get it to work.

If I want a simple home page with my content, my footer and so. Do I have to create a collection with for exemple "Content" entry and "Footer" entry?

**Reply by jasonvarga** [reply]
It's only automatic on Statamic routes. You're creating a custom route with a custom controller, so you have to do it yourself.

What doesn't work? Get the id of the entry from the markdown file, and do `Entry::find('the-id')`. What do you get?

If you want the template-into-layout feature, then instead of Laravel's `view()` function, you can use this:

```php
use Statamic\View\View;

return View::make('home.home')->layout('layout')->with($data);
```
